official/utils/arg_parsers/parsers.py

```diff
@@ -151,6 +152,7 @@ def __init__(self, add_help=False, num_parallel_calls=True, inter_op=True,
                intra_op=True, use_synthetic_data=True, max_train_steps=True):
     super(PerformanceParser, self).__init__(add_help=add_help)
 
+    # TODO(taylorrobie@): deprecate and only use DistributionStrategies
```

Comment:
Right now, only MNIST uses, correct? Can you follow up with making that use DistStrat as well? And, in theory, WideDeep should be easy, because it's just estimators... is that true? Can you check and update as well if so?